src/containers/ButtonsContainer.js

```diff
@@ -0,0 +1,24 @@
+import { connect } from 'react-redux';
+import { addTodo } from '../actions';
+import ButtonsComponent from '../components/ButtonsComponent.jsx';
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    Click: (text, importance) => {
+      dispatch(addTodo(text, importance))
```

Comment:
https://redux.js.org/docs/api/bindActionCreators.html can be used to bind action creators